test/views/ImporterVatRegisteredViewSpec.scala

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import base.ViewBaseSpec
+import forms.ImporterVatRegisteredFormProvider
+import messages.{BaseMessages, ImporterVatRegisteredMessages}
+import org.jsoup.Jsoup
+import org.jsoup.nodes.Document
+import play.api.data.Form
+import play.twirl.api.Html
+import views.html.ImporterVatRegisteredView
+
+class ImporterVatRegisteredViewSpec extends ViewBaseSpec with BaseMessages {
+
+  private lazy val injectedView: ImporterVatRegisteredView = app.injector.instanceOf[ImporterVatRegisteredView]
+
+  val formProvider: ImporterVatRegisteredFormProvider = injector.instanceOf[ImporterVatRegisteredFormProvider]
+
+  "Rendering the ImporterVATRegistered page" when {
+    "no errors exist" should {
+
+      val form: Form[Boolean] = formProvider.apply()
+      lazy val view: Html = injectedView(form)(fakeRequest, messages)
+      lazy implicit val document: Document = Jsoup.parse(view.body)
+
+      s"have the correct page title" in {
+        document.title mustBe ImporterVatRegisteredMessages.title
+      }
+
+      "not render an error summary" in {
+        document.select("div.govuk-error-summary").size mustBe 0
+      }
+
+      "not render an error message against the field" in {
+        document.select("#value-error").size mustBe 0
+      }
+    }
+
+    "an error exists (no option has been selected)" should {
+      lazy val form: Form[Boolean] = formProvider().bind(Map("value" -> ""))
+      lazy val view: Html = injectedView(form)(fakeRequest, messages)
+      lazy implicit val document: Document = Jsoup.parse(view.body)
+
+      "update the page title to include the error prefix" in {
+        document.title mustBe ImporterVatRegisteredMessages.errorPrefix + ImporterVatRegisteredMessages.title
```

Comment:
```suggestion
        document.title mustBe errorPrefix + ImporterVatRegisteredMessages.title
```

Reply:
for errorPrefix, siteNo, siteYes you can remove the ImporterVatRegisteredMessages and it will still work as the view spec is extending the base messages so you gain access to them without the need to use the full name of object and value